tailormap-components/projects/bridge/typings/Components.d.ts

```diff
@@ -27,6 +27,9 @@ declare interface ViewerController {
   getAppLayerById: (appLayerId: number) => AppLayer;
   getAppLayer: (serviceId: number, layerName: string) => AppLayer;
 
-  getVisibleLayers: (castToStrings?: boolean) => number[] | string[];
+  getVisibleLayers(): number[];
```

Comment:
By using function overloads you can define the return type for different scenarios

Reply:
Well, now I feel stupid...